modules/programs/mbsync.nix

```diff
@@ -10,6 +10,24 @@ let
   mbsyncAccounts =
     filter (a: a.mbsync.enable) (attrValues config.accounts.email.accounts);
 
+  # Given a SINGLE group's channels attribute set, return true if ANY of the channel's
+  # patterns use the invalidOption attribute set value name.
+  channelInvalidOption = channels: invalidOption:
+    any (c: c == true) (mapAttrsToList (c: hasAttr invalidOption) channels);
```

Comment:
Just switched to `(c: c)`. If that fails to test properly, then I will change it back. I feel the `anyTrue` function might be more effective as a library function, because I think it would be useful in more places than this.